doc/whats_new/v1.5.rst

```diff
@@ -47,6 +47,12 @@ See :ref:`array_api` for more details.
 
 **Classes:**
 
+**Libraries:**
+
+- ``dask.array`` is now experimentally supported as an array API backend.
```

Comment:
```suggestion
- ``dask.array`` is now experimentally supported as an Array API backend.
```

Reply:
Looks like Olivier commented on a few of these before as well. I won't comment all that I find, but it would be great to have "Array API" every where.

Reply:
not to be pedantic, but I would recommend _not_ capitalising this as per https://github.com/data-apis/array-api/discussions/778 !

Reply:
Thanks for the feedback, I was not aware of this. I guess we have a lot of doc/comments to update...

Let's fix that in a dedicated PR.